config/locales/en.yml

```diff
@@ -81,6 +81,12 @@ en:
   invalid_credentials: Login failed due to invalid credentials. Are you sure you entered them correctly?
   invalid_login_method: Login failed due to account mismatch. You need to log in with omniauth.
   invite_message: "To invite someone to the meeting, send them this link:"
+  javascript:
+    room:
+      mailer:
+        subject: 'has invited you to view a recording.'
+        body: 'To view the recording, follow the link below:\n\n'
```

Comment:
I think it might be better to move the \n to the javascript file (since it doesn't need to be translated)

Reply:
Agreed